pyccel/ast/class_defs.py

```diff
@@ -130,6 +135,14 @@
 
 #=======================================================================================
 
+ListClass = ClassDef('list', class_type = NativeHomogeneousList(),
+        methods=[
+            PyccelFunctionDef('append', func_class = ListAppend,
+                decorators = {'property': 'property'}),
```

Comment:
Sorry, I was just talking about the property decorator and remembered this was here. Why is this needed? Doesn't this cause a TypeError if you try to translate
```python
a = [1,2,3]
a.append
```

Reply:
I specified it cs the `ListAppend` class uses it 
I didn't quite understand what you meant with this example ?

Reply:
Can you point me at where the `ListAppend` class uses the `property` decorator please. 

The property decorator (https://www.geeksforgeeks.org/python-property-decorator-property/) is used to allow a function to be called without passing brackets. So normally indicating that the `PyccelFunctionDef` has the `property` decorator should be the same as saying that you are allowed to call the function with the following syntax:
```
a = [1,2,3]
a.append
```
Without the decorator the code above will raise a neat error saying that the property is unknown. With the decorator it will try to call `ListAppend` passing only the list as an argument

Reply:
Ohh my bad, i thought it was about the decorator that's used for attributes, Python's beginners' mistake.